app.js

```diff
@@ -37,6 +49,7 @@ app.use((req, res, next) => {
 // Routes
 app.use("/", indexRoute)
 app.use("/listings", listingRoute)
+app.use("/", userRoute)
```

Comment:
two routes on same api prefix? indexRoute and userRoute both on "/" ?? we can give this route as "auth"

Reply:
done